src/modules/file/file.repository.ts

```diff
@@ -151,6 +152,21 @@ export class SequelizeFileRepository implements FileRepository {
     @InjectModel(FileModel)
     private fileModel: typeof FileModel,
   ) {}
+  async deleteByFileId(deleteByFileId: string): Promise<void> {
```

Comment:
Many tests call this function, I made a throw error implementation to prevent the tests from giving errors, if you require in another PR I can clean all the tests from this function.